frontend/src/scenes/data-warehouse/external/forms/SyncProgressStep.tsx

```diff
@@ -43,6 +43,63 @@ export const SyncProgressStep = (): JSX.Element => {
         }
     }
 
+    const columns: LemonTableColumns<ExternalDataSourceSchema> = [
+        {
+            title: 'Table',
+            key: 'table',
+            render: function RenderTable(_, schema) {
+                return schema.name
+            },
+        },
+        {
+            title: 'Status',
+            key: 'status',
+            render: function RenderStatus(_, schema) {
+                const { status, tagType } = getSyncStatus(schema)
+
+                return <LemonTag type={tagType}>{status}</LemonTag>
+            },
+        },
+    ]
+
+    if (!isWrapped) {
+        columns.push({
+            key: 'actions',
+            width: 0,
+            render: function RenderStatus(_, schema) {
+                if (schema.table && schema.status === 'Completed') {
+                    const query: DataTableNode = {
+                        kind: NodeKind.DataTableNode,
+                        full: true,
+                        source: {
+                            kind: NodeKind.HogQLQuery,
+                            query: `SELECT ${schema.table.columns
+                                .filter(
+                                    ({ table, fields, chain, schema_valid }) =>
+                                        !table && !fields && !chain && schema_valid
+                                )
+                                .map(({ name }) => name)} FROM ${
+                                schema.table.name === 'numbers' ? 'numbers(0, 10)' : schema.table.name
+                            } LIMIT 100`,
+                        },
+                    }
```

Comment:
We've got this in a few places now (I think), would be good to make it into a util and reused